plugins/web-plugin/src/main/java/ru/sbtqa/tag/stepdefs/WebGenericSteps.java

```diff
@@ -8,8 +8,13 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import ru.sbtqa.tag.pagefactory.PageManager;
+import ru.sbtqa.tag.pagefactory.annotations.ActionTitle;
+import ru.sbtqa.tag.pagefactory.annotations.ActionTitles;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'ru.sbtqa.tag.pagefactory.annotations.ActionTitles'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=96&issues=AWZh47BsrFHbA8-Egpm0&open=AWZh47BsrFHbA8-Egpm0)

[ISSUE_KEY]: AWZh47BsrFHbA8-Egpm0